### PR TITLE
Update docker_compose.md [skip ci]

### LINKED ISSUE #1637: [DOCS] Extend Docker installation instructions (docs)
These instructions should clarify how to get started with Docker

### PR DESCRIPTION
**Pull request by Dumnersm580**
# Description

Removed emojis from the page and used words to express the same meaning
Rewrote the headings to be more descriptive
Replaced the exclamation marks with periods to make the sentences more professional
Removed contractions to make the text more formal
Rewrote some of the sentences to make them clearer and more concise
Added punctuation where necessary
Used bullet points to organize some of the information
Removed unnecessary abbreviations
Removed an unnecessary sentence that contained no information
Rewrote some of the sentences to avoid using colloquial expressions
Removed unnecessary words and phrases
Made the language more professional and understandable

Fixes #1637 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] This change requires a documentation update

### CODE REVIEW

File: docs/guides/docker_compose.md [hunk 1]
```diff
@@ -6,23 +6,21 @@ nav_order: 2
 last_modified_at: 2023-05-05T10:01:00+02:00
```
[the remark this answers is not in the document]
Comment (Dumnersm580) [reply docs/guides/docker_compose.md:6]: got it

File: docs/guides/docker_compose.md [hunk 2]
```diff
@@ -6,23 +6,21 @@ nav_order: 2
...
-> :warning: **This page is work in progress!**
+## Docker Compose for FDM Monster
 
-# Running FDM Monster with docker-compose
+This is a guide on how to use Docker Compose to run FDM Monster. Please note that this page is a work in progress.
 
-Docker is a great tool! Using `docker-compose` is better and `portainer` is most awesome! Please read the following before continuing:
-1) NOTE we assume you are familiar with `docker` and `docker-compose`. These are great tools for isolating your software deployment (+), but it be quite new to some users (-).
-    - We cannot support each custom scenario or setup!
-    - Take good care of checking your device's memory limits, architecture and CPU power (`docker stats` and for example `mem_limit: 400m` for 400MB limit in docker-compose)
```
Comment (davidzwa) [inline docs/guides/docker_compose.md:16]: The 400MB condition should be removed as the flag is not supported in compose v3+. Same goes for the actualy .yml content below.

**Comment by davidzwa** [comment]
@all-contributors add @Dumnersm580 for doc

**Review by davidzwa — APPROVED** [review]
LGTM

**Review by Dumnersm580 — COMMENTED** [review]
done

**Comment by davidzwa** [comment]
@all-contributors add @Dumnersm580 for ideas

**Pull request merged**